cpp/ql/src/jsf/4.10 Classes/AV Rule 79.ql

```diff
@@ -110,7 +111,8 @@ private predicate exprReleases(Expr e, Expr released, string kind) {
     ) and
     e.(FunctionCall).getQualifier() = exprOrDereference(released) and
     innerThis.getEnclosingFunction() = f and
-    exprReleases(_, innerThis, kind)
+    globalValueNumber(innerThis) = globalValueNumber(likeInnerThis) and
+    exprReleases(_, likeInnerThis, kind)
```

Comment:
You could avoid declaring `likeInnerThis` by using `globalValueNumber(innerThis).getAnExpr()`.

Reply:
Good point! Fixed in 2abf4c068f79785126a617a40b048c064b786f36.